Automatic Validation ended with: 
> 2026-04-15T11:11:09.3567796Z ##[error] Manifest Error: Field usage requires verified publishers. [Icons]

(Automated response - build 1248.)

--- a/manifests/v/VibeSmiths/HeatSync/1.0.76/VibeSmiths.HeatSync.installer.yaml
+++ b/manifests/v/VibeSmiths/HeatSync/1.0.76/VibeSmiths.HeatSync.installer.yaml
@@ -12,6 +12,6 @@ Dependencies:
 Installers:
 - Architecture: x64
   InstallerUrl: https://github.com/VibeSmiths/HeatSync/releases/download/v1.0.76/HeatSync.exe
-  InstallerSha256: 5A00EFBF9144A92AFE7F9A053510E481497E89E71A1E105EA208ECB017973323
+  InstallerSha256: DCB36C7A649B90FAC379EBB1E9700F066D585EF15345FCBD4C3959C010B0611B
 ManifestType: installer
 ManifestVersion: 1.12.0

--- a/manifests/v/VibeSmiths/HeatSync/1.0.76/VibeSmiths.HeatSync.installer.yaml
+++ b/manifests/v/VibeSmiths/HeatSync/1.0.76/VibeSmiths.HeatSync.installer.yaml
@@ -6,6 +6,9 @@ PackageVersion: 1.0.76
 InstallerType: portable
 Commands:
 - heatsync
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: namazso.PawnIO
 Installers:
 - Architecture: x64
   InstallerUrl: https://github.com/VibeSmiths/HeatSync/releases/download/v1.0.76/HeatSync.exe

--- a/manifests/v/VibeSmiths/HeatSync/1.0.76/VibeSmiths.HeatSync.locale.en-US.yaml
+++ b/manifests/v/VibeSmiths/HeatSync/1.0.76/VibeSmiths.HeatSync.locale.en-US.yaml
@@ -29,9 +29,5 @@ InstallationNotes: 'Launch with: heatsync. UAC prompt appears on first run becau
 Documentations:
 - DocumentLabel: README
   DocumentUrl: https://github.com/VibeSmiths/HeatSync/blob/main/README.md
-Icons:
-- IconUrl: https://raw.githubusercontent.com/VibeSmiths/HeatSync/main/assets/icon.png
-  IconFileType: png
-  IconResolution: 256x256
 ManifestType: defaultLocale
 ManifestVersion: 1.12.0

